packages/sdk-server/src/sdk-client.ts

```diff
@@ -115,10 +116,16 @@ export class TurnkeyServerSDK {
   };
 }
 
-export class TurnkeySDKServerClient extends TurnkeySDKClientBase {
+export class TurnkeyServerClient extends TurnkeySDKClientBase {
```

Comment:
is there anything that explicitly uses `TurnkeyServerClient` now that we also have `TurnkeyApiClient`?

Reply:
no, but this is just modeling the same format as the browser sdk with the intermediary BrowserClient and ServerClient for the individual clients to inherit from